src/UI/UITest/main.cpp

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
I was not sure how should I do this because ,testing part is in another project so , I made 2 pch.h files , one for UI and one for UITest

In pch.h from UITest I had included pch.h from UI.

In VS pch.cpp is needed but in qt there is no information about that file so i think it is not needed.

In .h files Should I just include external headers like before ? 

I made benchmarks for two files and that's the result :

One file edit in UI
With pch 0.07s
Without pch 0.08s

One file edit in UITest
With pch 0.03s
Without pch 0.04s

Reply:
I separated two pch.h files